angularjs-portal-home/src/main/webapp/my-app/layout/widget/controllers.js

```diff
@@ -4,6 +4,9 @@ define(['angular'], function(angular){
 
   var app = angular.module('my-app.layout.widget.controllers', []);
 
+  app.controller('ListOfLinksController', ['$scope', 'layoutService', function($scope, layoutService){
+  }]);
```

Comment:
if there is nothing to do in the controller, you don't need one true? (directives don't have to have a controller connected to them)